cdc/log.cc

```diff
@@ -68,6 +69,70 @@ namespace cdc {
 static schema_ptr create_log_schema(const schema&, std::optional<utils::UUID> = {});
 }
 
+static constexpr auto cdc_group_name = "cdc";
+
+void cdc::stats::parts_touched_stats::register_metrics(seastar::metrics::metric_groups& metrics, sstring suffix) {
+    namespace sm = seastar::metrics;
+    auto register_part = [&] (part_type part, sm::label_instance part_instance) {
+        metrics.add_group(cdc_group_name, {
+                sm::make_total_operations("components_touched_" + suffix, count[(size_t)part],
```

Comment:
isn't the suffix equal to the label?
if so, it's redundant.
you should either differ by name or by label, not both

Reply:
They are different. The suffix is either "total" or "failed". The label determines which part of the mutation was touched (e.g. "static_row", "row_tombstone", "udt", etc.).

Reply:
In my opinion, operations_performed would be better than components_touched